ietf/templates/doc/document_info.html

```diff
@@ -431,7 +431,7 @@
                         {% endfor %}
                         {% if doc.group and doc.group.list_archive %}
                             {% if doc.group.list_archive|startswith:settings.MAILING_LIST_ARCHIVE_URL %}
-                                <a href="{{ doc.group.list_archive }}?q={{ doc.name }}">
+                                <a href="{{ doc.group.list_archive }}?q={{ doc.name }}{% if doc.came_from_draft %} OR %22{{ doc.came_from_draft.name }}%22{% endif%}">
```

Comment:
Why not URL encode spaces around `OR` as well?

Reply:
Can you find an ecosystem where it's actually necessary? (escaping the double-quotes was mostly a quick way to construct a string that django wouldn't mangle on the way out).

Reply:
I requested a change just to avoid confusion. It's not apparent that spaces around `OR` are important.